sanket/javascript/assignment12(day25)/sanketA12_Q1.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>employee details</title>
+  </head>
+  <body>
+    <script>
+      class Employee {
+        constructor() {
+          this.empNo = 101;
+          this.name = "Ashok Kumar";
+          this.address = "43 Ramnagar";
+          this.city = "Pune Maharashtra";
+          this.salary = 80000;
+        }
+        showInit() {
+          document.write(
+            `employeeNo= ${this.empNo}, name= ${this.name}, address= ${this.address},
+            city= ${this.city}, salary= ${this.salary}<br>`
+          );
+        }
+        calculate() {
+          let da = 0.3 * this.salary;
+          let hra = 0.15 * this.salary;
+          let pf = 0.125 * this.salary;
+          this.gross = this.salary + da + hra;
```

Comment:
remove this.salary and add basic salary like: this.basic = this.salary * 12
```suggestion
          this.basic = this.salary *12;
          this.gross = (this.basic) + da + hra;
```

Reply:
i think you are not getting the question, please read the question properly also, salary and basic salary, annual basic salary

Reply:
Ok

Reply:
this.gross = this.salary + da + hra;
the previous statement is correct